packages/db/src/types.ts

```diff
@@ -1,52 +1,52 @@
 import { AsyncOperationStatus, AsyncOperationType } from '@cumulus/types/api/async_operations';
```

Comment:
I saw inconsistency in this file and opted to add commas, but I think we need to discuss if we want commas or nothing.